src/app/lib/layoutGrid.js

```diff
@@ -27,24 +27,36 @@ const group4WrapperMaxWidth = `45.5rem`;
 const group5WrapperMaxWidth = `46.4rem`;
 // (2.95rem * 12) + 11*16px gutters = 742.4 = 46.4 rem
 
-const fiveOfSixColumnsMaxWidthGroup4 = `37.75rem`;
+const starts3spans5of8MaxWidthGroup4 = `37.75rem`;
```

Comment:
Im not a fan of this renaming `37.75rem` doesnt need to start 3 columns it, it could be starts4spans5of8 and be equally functional

Reply:
```suggestion
const spans5of8MaxWidthGroup4 = `37.75rem`;
```
so this would be better? @dr3